packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -43,15 +44,19 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
       }, [context, disabled, itemKey, onClick]
     );
 
-    const onKeyUp = React.useCallback(
-      e => {
-        if (e.nativeEvent.key === 'Enter' || e.nativeEvent.key === ' ') {
-          onItemClick(e);
-        }
-      }, [onItemClick]);
+    const cmRef = useViewCommandFocus(componentRef);
+    //const cmRef = React.useRef(null);
+    const onItemHoverIn = React.useCallback(
+      () => {
+        componentRef.current.focus();
+      }, [componentRef]);
 
-    // attach the pressable state handlers
-    const pressable = useAsPressable({ ...rest, onPress: onItemClick });
+    const onItemHoverOut = React.useCallback(
+      () => {
+        componentRef.current.blur();
```

Comment:
I'm not sure I'd bother with the blur call.  The next item to take focus will remove focus from the previous item.

Reply:
Got it!